Doc/c-api/refcounting.rst

```diff
@@ -109,11 +109,17 @@ objects.
    It is a good idea to use this macro whenever decrementing the reference
    count of an object that might be traversed during garbage collection.
 
+.. c:function:: void Py_IncRef(PyObject *o)
+
+   :c:func:`Py_IncRef` is a simply exported function version of :c:func:`Py_XINCREF`,
+   which provides conveniences for runtime dynamic embedding of Python.
```

Comment:
```suggestion
   Increment the reference count for object *o*. A function version of :c:func:`Py_XINCREF`.
   It can be used for runtime dynamic embedding of Python.
```
- Say what it does
- The "simply exported" wording doesn't make sense here
- Tighten up wording